pkg/virt-handler/vm.go

```diff
@@ -2424,17 +2425,17 @@ func (d *VirtualMachineController) updateNodeCpuManagerLabel(cpuManagerPath stri
 
 }
 
-func (d *VirtualMachineController) setVMIGuestTime(vmi *v1.VirtualMachineInstance) error {
-	// update the vmi guest with the current time
+func (d *VirtualMachineController) finalizeMigration(vmi *v1.VirtualMachineInstance) error {
 	client, err := d.getVerifiedLauncherClient(vmi)
 	if err != nil {
 		return err
 	}
-	err = client.SetVirtualMachineGuestTime(vmi)
+	err = client.FinalizeVirtualMachineMigration(vmi)
 	if err != nil {
-		log.Log.Reason(err).Error("failed to set vmi guest time to the current")
+		log.Log.Reason(err).Error("failed to finalize migration")
```

Comment:
Can you also add the `vmi` object so the log will include the reference to it.
(I mean `log.Log.Object(vmi).Reason(err)`)

You can do this in a separate commit, up to you.

Reply:
Done